sslSniffer/sslSnifferTest/snifftest.c

```diff
@@ -569,6 +580,320 @@ static int SnifferAsyncPollQueue(byte** data, char* err, SSLInfo* sslInfo,
 }
 #endif /* WOLFSSL_ASYNC_CRYPT */
 
+#ifdef THREADED_SNIFFTEST
+
+/* Change to increase number of threads */
+const int workerThreadCount = 3;
```

Comment:
Please make this an overridable build macro:
```
#ifndef THREADED_SNIFFTEST_WORKERS
#define THREADED_SNIFFTEST_WORKERS 3
#endif

static const int workThreadCount = THREADED_SNIFFTEST_WORKERS;
```

Reply:
Done